components/tidb_query_expr/src/impl_string.rs

```diff
@@ -825,6 +912,8 @@ mod tests {
 
     use crate::types::test_util::RpnFnScalarEvaluator;
 
+    use tidb_query_datatype::codec::mysql::MAX_FSP;
```

Comment:
Please rearrange imports into this form:

```rust
use std::

use tipb::
use other_crates::

use crate::
use super::
```

Reply:
Okay~

done